extensions/wikia/AdEngine/js/slot/bottomLeaderboard.js

```diff
@@ -0,0 +1,44 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.bottomLeaderboard', [
+	'ext.wikia.adEngine.adHelper',
+	'ext.wikia.adEngine.provider.btfBlocker',
+	'ext.wikia.adEngine.utils.domCalculator',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window',
+], function (adHelper, btfBlocker, dom, doc, log, win) {
+	'use strict';
+
+	var slotName = 'BOTTOM_LEADERBOARD',
+		mainContent = doc.getElementById('WikiaMainContent'),
+		threshold = 100,
+		viewPortHeight = Math.max(doc.documentElement.clientHeight, win.innerHeight || 0),
+		logGroup = 'ext.wikia.adEngine.slot.bottomLeaderboard',
+		pushed = false,
+
+		pushBfab = adHelper.throttle(function () {
+			var scrollPosition = win.scrollY || win.pageYOffset || doc.documentElement.scrollTop,
+				pushPos = dom.getTopOffset(mainContent) + mainContent.offsetHeight - viewPortHeight - threshold;
+
+			if (!pushed && pushPos < scrollPosition) {
+				pushed = true;
+				doc.removeEventListener('scroll', pushBfab);
+				btfBlocker.unblock(slotName);
```

Comment:
`BTF` slots are not run in `BFAA` yet -- so if I unblock `BOTTOM_LEADERBOARD` there it won't be pushed on scroll but much earlier.